README.md

```diff
@@ -20,7 +20,11 @@ npm run dev
 # Typechecking (via TypeScript / tsc)
 npm run typecheck
 
-# Linting (via eslint)
```

Comment:
what happened to eslint doc? 😢

Reply:
i thought the command was removed but someone forgot to remove the comment as well